tinywallet/pyproject.toml

```diff
@@ -14,7 +14,7 @@ classifiers = [
     "Development Status :: 3 - Alpha",
     "Environment :: X11 Applications :: Qt",
     "Intended Audience :: End Users/Desktop",
-    "License :: OSI Approved :: ISC License (ISCL)",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
```

Comment:
We may not have to change our license. Check out the source code for **PySide2**. There is a file called *LICENSE.GPLv3-EXCEPT* which says

> As a special exception you may create a larger work which contains the output of this application and distribute that work under terms of your choice, so long as the work is not otherwise derived from or based on this application and so long as the work does not in itself generate output that contains the output from this application in its original or modified form.

Reply:
IANAL, but this text seems to be geared toward an application or tool rather than a library, so much that it's weird finding it as part of a library.

TinyWallet clearly seems based on PySide2, so I don't think this exception applies anyway.

Reply:
> TinyWallet clearly seems based on PySide2

I'm not sure exactly what that means. Does "based on" have a particular definition that you're using? TinyWallet offers no GUI tookit utilities. 

> IANAL

Me neither, which is just as much of an argument to not change the license. But regardless, I did read the LGPLv3, and it seems clear that we are an `Application`, that together with the `Library` form a `Combined Work` which we are free to distribute under our terms as long as we don't modify the terms of the `Library`'s license (see section 4). We of course must leave the license file, but PyPI already distributes that for us.

The only place that I'm not sure that we're compliant is where it says. 

> Give prominent notice with each copy of the Combined Work that the Library is used in it and that the Library and its use are covered by this License.

Reply:
> I'm not sure exactly what that means. Does "based on" have a particular definition that you're using?

I understand "derived work" and "based on" as something that could not exist without the library it is based on. This is debatable given the existence of PyQt5 with the same API, but that has a more restrictive license (GPL).

> it seems clear that we are an Application, that together with the Library form a Combined Work which we are free to distribute under our terms

You're probably right. The FSF [says](https://www.gnu.org/licenses/why-not-lgpl.html):

"using the Lesser GPL permits use of the library in proprietary programs" and if we can do that we can also use a different, much less restrictive license.

> The only place that I'm not sure that we're compliant is where it says:

I guess mentioning PySide2 and its license in TinyWallet's README should be prominent enough. We can mention them in the main README too, to be on the safe side.